coordinator/ethereum/gas-pricing/static-cap/src/main/kotlin/net/consensys/linea/ethereum/gaspricing/staticcap/FeeHistoryFetcherImpl.kt

```diff
@@ -31,7 +31,7 @@ class FeeHistoryFetcherImpl(
   }
 
   private var cacheIsValidForBlockNumber: BigInteger = BigInteger.ZERO
-  private var feesCache: FeeHistory = getRecentFees().get()
+  private lateinit var feesCache: FeeHistory
```

Comment:
To avoid calling `getRecentFees().get()` immediately on coordinator start-up (i.e. when L1 current ethBlockNumber is zero) which would lead to `get(...) must not be null` NPE